src/Nest/CommonAbstractions/ConnectionSettings/ConnectionSettingsBase.cs

```diff
@@ -134,10 +125,14 @@ public TConnectionSettings DefaultIndex(string defaultIndex)
 			return (TConnectionSettings) this;
 		}
 
-		private string LowerCaseAndPluralizeTypeNameInferrer(Type type)
+		/// <summary>
+		/// A default Elasticsearch type name to use for CLR types. If you also set <see cref="DefaultTypeNameInferrer"/> this will only take affect if your callback
```

Comment:
Took me more than a casual glance to understand what this comment was saying (_it is Monday morning though!_); I think it could be a little clearer. Maybe something like
```
Sets a default type name to use within Elasticsearch for all CLR types. If <see cref="DefaultTypeNameInferrer"/> is also set, a configured default type name will only be used when <see cref="DefaultTypeNameInferrer"/>returns null or empty. If unset, the default type name for types will be the lowercased CLR type name.
```
?